### DISCUSSION TITLE
Мгновенное открытие окна загрузки по нажатию кнопки

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#447] [Как это сделать?] [answered]
Хотел предложить такие штуки, чтобы окно загрузки файлов сразу открывалось:

1. Добавить возможность однаразового клика на поле Файл, чтобы открывалось окно загрузки файлов.

2. Добавить какие-то параметры, чтобы по нажатию Кнопки, можно было открыть окно загрузки файлов, минуя множественные клики.

Закинул примеры видео и скрин.
Видео:
[Файлы — ERGO.webm](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/ef1098e1-0c9e-4d95-92fc-ebe15a3ea170)

Скрин:
![Screenshot_6](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/e95d077d-3ae3-428e-923d-db9476933dcb)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Поменял категорию, тк меня сбило предложенное:

1. Попробуйте поставить параметра pointing: в linktodatatable — тогда фокус будет в поле и останется нажать пробел

2. Попробуйте сначала вызвать linktofileupload и потом уже открыть что-то типа linktoinput для внесения описания и потом записать уже в табл

Тут только сложное может бытьпередавать коды в каскад инпутов — это делается через переменные.
